pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -172,6 +173,7 @@ def spawn(self, function: Callable, *args: Any, return_result: bool = True, **kw
         Return:
             The output of the function of process 0.
         """
+        self._clean_logger(*args)
```

Comment:
if this fix works, I propose to do it in the `start_x` methods above, before calling spawn().

Reply:
Done !